client/service/src/task_manager.rs

```diff
@@ -158,27 +155,27 @@ impl TaskManager {
 		}
 	}
 
-	pub(crate) fn spawn_handle(&self) -> SpawnTaskHandle {
+	pub fn spawn_handle(&self) -> SpawnTaskHandle {
 		SpawnTaskHandle {
 			on_exit: self.on_exit.clone(),
 			sender: self.to_spawn_tx.clone(),
 		}
 	}
 
 	/// Get sender where background/async tasks can be sent.
-	pub(crate) fn scheduler(&self) -> TaskScheduler {
+	pub fn scheduler(&self) -> TaskScheduler {
 		self.to_spawn_tx.clone()
 	}
 
 	/// Process background task receiver.
-	pub(crate) fn process_receiver(&mut self, cx: &mut Context) {
+	pub fn process_receiver(&mut self, cx: &mut Context) {
```

Comment:
TaskManager is now not exposed and is internal within the `sc_service` crate

Reply:
I don't know, I'd still mark these functions as `pub(super)`. I can totally imagine someone sneaking in a pull request exposing the task manager without thinking about it, and then we're stuck with this PR.